src/pages/marketplace/NFTs/CreateNFTPage.vue

```diff
@@ -75,7 +75,7 @@ export default {
     },
     async uploadToPrivateService ({ attributes, addressToShare }) {
```

Comment:
I think we should remove all methods related with private api

Reply:
Sure! I will remove it from the components